ui/raidboss/data/05-shb/raid/e3s.js

```diff
@@ -180,14 +139,7 @@
       regexJa: Regexes.startsUsing({ id: '3FEE', source: 'リヴァイアサン', capture: false }),
       regexCn: Regexes.startsUsing({ id: '3FEE', source: '利维亚桑', capture: false }),
       regexKo: Regexes.startsUsing({ id: '3FEE', source: '리바이어선', capture: false }),
-      alarmText: {
-        en: 'Go Outside',
-        de: 'Geh nach Ausen',
-        fr: 'Allez sur les côtés',
-        ja: '中壊れるよ',
-        cn: '两侧',
-        ko: '양옆으로',
-      },
+      response: Responses.getOut(),
```

Comment:
This should really be go sides (I know that's not what it said before), and also an alarm